scopes/generator/generator/generator.docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: Generator for generating component templates
+description: Generates components from templates, for more efficient development of new components that adhere to best-practices and organization standards.
```

Comment:
A simple interface for generating component templates.

Reply:
done